admin/move_attachments.php

```diff
@@ -193,55 +199,58 @@ function move_attachments_to_disk( $p_type, array $p_projects ) {
 			$t_failures = 0;
 			$t_data = array();
 
-			while( $t_row = db_fetch_array( $t_result ) ) {
-				$t_disk_filename = $t_upload_path . $t_row['diskfile'];
-				if ( file_exists( $t_disk_filename ) ) {
-					$t_status = 'Disk File Already Exists \'' . $t_disk_filename . '\'';
-					$t_failures++;
-				} else {
-					# write file to disk
-					if( file_put_contents( $t_disk_filename, $t_row['content'] ) ) {
-						# successful, update database
-						# @todo do we want to check the size of data transfer matches here?
-						switch( $p_type ) {
-							case 'project':
-								$t_update_query = 'UPDATE {project_file}
-									SET folder = ' . db_param() . ', content = \'\'
-									WHERE id = ' . db_param();
-								break;
-							case 'bug':
-								$t_update_query = 'UPDATE {bug_file}
-									SET folder = ' . db_param() . ', content = \'\'
-									WHERE id = ' . db_param();
-								break;
-						}
-						$t_update_result = db_query(
-							$t_update_query,
-							array( $t_upload_path, $t_row['id'] )
-						);
-
-						if( !$t_update_result ) {
-							$t_status = 'Database update failed';
-							$t_failures++;
+			while( $t_row2 = db_fetch_array( $t_result ) ) {
+				$t_result2 = db_query( $t_query2, array( $t_row2['id'] ) );
+				while( $t_row = db_fetch_array( $t_result2 ) ) {
```

Comment:
Variable naming is confusing - you're using $t_row to process the results of $t_query2/$t_result2 and $t_row2 for $t_query/$t_result. I suggest to either refactor to use more meaningful names, or at the very least switch usage of $t_row/$t_row2 to match the corresponding query.